From 518916065b56d95af390ac2d2d165dffdb9887d6 Mon Sep 17 00:00:00 2001
From: Richard Levitte <levitte@openssl.org>
Date: Mon, 1 Jul 2024 15:28:39 +0200
Subject: [PATCH 048/125] fix: remove some odd empty lines

Reviewed-by: Neil Horman <nhorman@openssl.org>
Reviewed-by: Tom Cosgrove <tom.cosgrove@arm.com>
Reviewed-by: Tomas Mraz <tomas@openssl.org>
(Merged from https://github.com/openssl/openssl/pull/24776)

(cherry picked from commit cfe0bbdecadaebc6ad7ba5a3335b7a03522c434f)
---
 apps/speed.c | 2 --
 1 file changed, 2 deletions(-)

diff --git a/apps/speed.c b/apps/speed.c
index f9cbc4106c..24c6d08084 100644
--- a/apps/speed.c
+++ b/apps/speed.c
@@ -3037,7 +3037,6 @@ skip_hmac:
             loopargs[i].sigsize = loopargs[i].buflen;
             if (loopargs[i].dsa_sign_ctx[testnum] == NULL
                 || EVP_PKEY_sign_init(loopargs[i].dsa_sign_ctx[testnum]) <= 0
-
                 || EVP_PKEY_sign(loopargs[i].dsa_sign_ctx[testnum],
                                  loopargs[i].buf2,
                                  &loopargs[i].sigsize,
@@ -3114,7 +3113,6 @@ skip_hmac:
             loopargs[i].sigsize = loopargs[i].buflen;
             if (loopargs[i].ecdsa_sign_ctx[testnum] == NULL
                 || EVP_PKEY_sign_init(loopargs[i].ecdsa_sign_ctx[testnum]) <= 0
-
                 || EVP_PKEY_sign(loopargs[i].ecdsa_sign_ctx[testnum],
                                  loopargs[i].buf2,
                                  &loopargs[i].sigsize,
-- 
2.39.5 (Apple Git-154)

